src/null-safety/unsound-null-safety.md

```diff
@@ -150,11 +150,13 @@ To test or run mixed-version code,
 you need to disable sound null safety.
 You can do this in two ways:
 
-* Disable sound null safety using the `--no-sound-null-safety` flag.
+* Disable sound null safety using the `--no-sound-null-safety` flag
+  to the `dart` or `flutter` command, at the command line or in your IDE.
```

Comment:
I would skip the "in your IDE" part. It's really hard to get flags right there. For IDEs I think the language version comment is a lot easier.